packages/launchpad/src/setup/TestingTypeCards.cy.tsx

```diff
@@ -82,4 +82,14 @@ describe('TestingTypeCards', () => {
     cy.findAllByText(defaultMessages.setupPage.testingCard.running).should('have.length', 1)
     cy.percySnapshot()
   })
+
+  it('renders beta label for component testing', () => {
+    cy.mountFragment(TestingTypeCardsFragmentDoc, {
+      render: (gqlVal) => {
+        return <TestingTypeCards gql={gqlVal} />
+      },
+    })
+
+    cy.get('[data-cy-testingtype="component"]').contains('Beta')
```

Comment:
Let's get a Percy snapshot here.

Reply:
Will do, and will re-ping for a review in around 14 hours from the time of this post.